nixos/lib/make-options-doc/mergeJSON.py

```diff
@@ -91,11 +91,11 @@ def link(self, link, text=None, title=None):
     def list(self, text, ordered, level, start=None):
         if ordered:
             raise NotImplementedError("ordered lists not supported yet")
-        return f"<itemizedlist>\n{text}\n</itemizedlist>"
+        return f"<para><itemizedlist>\n{text}\n</itemizedlist></para>"
     def list_item(self, text, level):
-        return f"<listitem><para>{text}</para></listitem>\n"
+        return f"<listitem>{text}</listitem>\n"
```

Comment:
How do we ensure the content is a valid child? The DocBook model does not allow text elements directly inside [`listitem`](https://tdg.docbook.org/tdg/5.1/listitem.html) and I assume Markdown parser will only insert paragraphs when there are multiple paragraphs in the list item or there are blank lines between the list items.